scripts/corstest.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# WF 2018-09-20
+# https://stackoverflow.com/a/47609921/1497139
+
+#ansi colors
+#http://www.csc.uvic.ca/~sae/seng265/fall04/tips/s265s047-tips/bash-using-colors.html
+blue='\033[0;34m'  
+red='\033[0;31m'  
+green='\033[0;32m' # '\e[1;32m' is too bright for white bg.
+endColor='\033[0m'
+
+#
+# a colored message 
+#   params:
+#     1: l_color - the color of the message
+#     2: l_msg - the message to display
+#
+color_msg() {
+  local l_color="$1"
+  local l_msg="$2"
+  echo -e "${l_color}$l_msg${endColor}"
+}
+
+
+#
+# show the usage
+#
+usage() {
+  echo "usage: [-v] $0 url"
+  echo "  -v |--verbose: show curl result" 
+  exit 1 
+}
+
+if [ $# -lt 1 ]
+then
+  usage
+fi
+
+# commandline option
+while [  "$1" != ""  ]
+do
+  url=$1
+  shift
+
+  # optionally show usage
+  case $url in      
+    -v|--verbose)
+       verbose=true;
+       ;;          
+  esac
+done  
```

Comment:
Do we need this if we call this script only without `-v` ?

```bash
while read URL; do
    $GITHUB_WORKSPACE/scripts/corstest.sh $URL
```

Reply:
Yes